src/js/views/patients/sidebar/action/action-sidebar-activity-views.js

```diff
@@ -96,6 +96,10 @@ const ActivityView = View.extend({
 
 const ActivitiesView = CollectionView.extend({
   childView: ActivityView,
+  viewFilter({ model }) {
+    if (model.get('type') === 'ActionCreated' && this.model.getProgram()) return false;
```

Comment:
for some reason I assumed we already had a `getProgram` when slacking earlier.  We probably won't need it for anything.. we can just use `get('_program')` here and add less code.